imap_processing/ena_maps/ena_maps.py

```diff
@@ -1315,7 +1315,7 @@ def build_cdf_dataset(
             if ("L2" in name)
         ]
         l2_coords.append(CoordNames.TIME.value)
-        for map_coord in cdf_ds.dims.keys():
+        for map_coord in cdf_ds.sizes.keys():
```

Comment:
I think you might be able to just get rid of the `.keys()` specifier here.
```suggestion
        for map_coord in cdf_ds.dims:
```